foundations/javascript_basics/project_calculator.md

```diff
@@ -1,46 +1,46 @@
-    - This is the hardest part of the project. You need to figure out how to store all the values and call the operate function with them. Don't feel bad if it takes you a while to figure out the logic.
+   - You should already have the code that can populate the display, so once `operate()` has been called, update the display with the 'solution' to the operation.
+   - This is the hardest part of the project. You need to figure out how to store all the values and call the operate function with them. Don't feel bad if it takes you a while to figure out the logic.
 1. Gotchas: watch out for and fix these bugs if they show up in your code:
-    - Users should be able to string together several operations and get the right answer, with each pair of numbers being evaluated at a time. For example, `12 + 7 - 5 * 3 =` should yield `42`. An example of the behavior we're looking for would be [this student solution](https://mrbuddh4.github.io/calculator/).
-    - **Your calculator should not evaluate more than a single pair of numbers at a time.** Example: you press a number button (`12`), followed by an operator button (`+`), a second number button (`7`), and finally a second operator button (`-`). Your calculator should then do the following: first, evaluate the first pair of numbers (`12 + 7`), second, display the result of that calculation (`19`), and finally, use that result (`19`) as the first number in your new calculation, along with the next operator (`-`).
-    - You should round answers with long decimals so that they don't overflow the screen.
-    - Pressing `=` before entering all of the numbers or an operator could cause problems!
-    - Pressing "clear" should wipe out any existing data.. make sure the user is really starting fresh after pressing "clear"
-    - Display a snarky error message if the user tries to divide by 0... and don't let it crash your calculator!
+   - **Your calculator should not evaluate more than a single pair of numbers at a time.** Example: you press a number button (`12`), followed by an operator button (`+`), a second number button (`7`), and finally a second operator button (`-`). Your calculator should then do the following: first, evaluate the first pair of numbers (`12 + 7`), second, display the result of that calculation (`19`), and finally, use that result (`19`) as the first number in your new calculation, along with the next operator (`-`). An example of the behavior we’re looking for can be seen in this [student's calculator live preview](https://mrbuddh4.github.io/calculator/).
+   - You should round answers with long decimals so that they don't overflow the screen.
+   - Pressing `=` before entering all of the numbers or an operator could cause problems!
+   - Pressing "clear" should wipe out any existing data.. make sure the user is really starting fresh after pressing "clear"
```

Comment:
```suggestion
   - Pressing "clear" should wipe out any existing data. Make sure the user is really starting fresh after pressing "clear".
```
Just noticed this pre-existing punctuation issue, if you wouldn't mind throwing this fix in as well?